Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -280,13 +280,21 @@ void OnScrollToRequested(object sender, ScrollToRequestedEventArgs e)
 			if (Element.IsGroupingEnabled)
 			{
 				var results = templatedItems.GetGroupAndIndexOfItem(scrollArgs.Group, scrollArgs.Item);
-				if (results.Item1 == -1 || results.Item2 == -1)
+				int indexOfGroup = results.Item1;
+				int indexOfItem = results.Item2;
+				
+				if (indexOfGroup == -1 || (indexOfGroup == -1 && indexOfItem == -1))
 					return;
 
-				var group = templatedItems.GetGroup(results.Item1);
-				cell = group[results.Item2];
+				int item = indexOfItem == -1 ? 0 : indexOfItem;
```

Comment:
Can we change the variable "item" to something more meaningful. It appears to be an item index?